### DISCUSSION TITLE
Ex 2 doesnt have joint position controller for EGM

### DISCUSSION THREAD

**Discussion by kalepo** [ros-industrial/abb_robot_driver#65] [Q&A]
Is there a joint position controller also for Example 2: "ex2_rws_and_egm_6axis_robot"?
I see a joint velocity controller in Example 2 but no joint position controller for EGM.

**Comment by gavanderhoorn** [comment]
> I see a joint velocity controller in Example 2 but no joint position controller for EGM.

Not sure what you mean by "but no joint position controller for EGM", but if you're asking how to load a `JointGroupPositionController`, you could simply change [this line](https://github.com/ros-industrial/abb_robot_driver/blob/32eb0f48abfd3358cb5a4f3f043f1306a9811b24/abb_robot_bringup_examples/config/ex2_controllers.yaml#L12) to read `position_controllers/JointGroupPositionController`.
